### DISCUSSION TITLE
Question 101:  Error message opening MM record [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#703] [Q&A] [answered]
Description: When I try to open any of the MM in groups IRN 15735 I get an error message that says: 

![101image](https://user-images.githubusercontent.com/702709/201959925-a54acbaa-5b6c-468c-a0ba-c6c7d0a67ad6.png)

This error message pops up, but then I am able to open the detail view for the record. This happens for all the records except MM irn 2413924, which I cannot open at all. Sorry I’m not able to make it today, I have a conflicting meeting.

**Comment by rondlg** [comment, marked as answer]
Response:
Could you check if you’re in the live emufmnh vs emutest environment? That’s generally an error message that pops up in test (where icons don’t display)

**Reply by zkachian** [reply]
I was in EMu live and getting that message. But I just reinstalled EMu and I'm no longer getting that message, so everything seems to be working properly now.
